src/mme-app/mmeStates/pagingStart.cpp

```diff
@@ -57,6 +57,12 @@ PagingStart* PagingStart::Instance()
 ******************************************************************************/
 void PagingStart::initialize()
 {
+        {
+                ActionTable actionTable;
+                actionTable.addAction(&ActionHandlers::send_paging_req_to_ue);
+                actionTable.setNextState(PagingWfServiceReq::Instance());
+                eventToActionsMap.insert(pair<uint16_t, ActionTable>(GW_CP_REQ_INIT_PAGING, actionTable));
```

Comment:
what is the difference between CBReq triggering PAGING vs DDN triggering PAGING

Reply:
We kept it as different entities, just to identify the procedure which triggers paging.